tests/integration/consumer/test_consumer_topicpartition_metadata.py

```diff
@@ -29,11 +29,11 @@ def commit_and_check(consumer, topic, metadata):
     assert offsets[0].metadata == metadata
 
 
-def test_consumer_topicpartition_metadata(kafka_cluster):
-    topic = kafka_cluster.create_topic("test_topicpartition")
+def test_consumer_topicpartition_metadata(kafka_single_broker_cluster):
+    topic = kafka_single_broker_cluster.create_topic("test_topicpartition")
```

Comment:
To make integration tests for admin APIs work with KRaft we need to add some sleeps after a number of operations:
`time.sleep(1)`
after:
- create_topic
- create_acls
- delete_acls
- alter_configs
- incremental_alter_configs
- alter_user_scram_credentials

Reply:
In `test_describe_operations `we can disable this test with `consumer`, as it returns it's a simple consumer group (needs to use the new RPC instead)
```Python
    if not TestUtils.use_group_protocol_consumer():
        verify_describe_groups(sasl_cluster, admin_client, our_topic)
```

Reply:
In `test_list_offsets` we need to disable this check at the moment
```python
        # FIXME: This test is disabled until KAFKA-16310 is released
        #        with AK 3.8.0
        # requests = {topic_partition: OffsetSpec.max_timestamp()}
        # futmap = admin_client.list_offsets(requests, **kwargs)
        # for _, fut in futmap.items():
        #     result = fut.result()
        #     assert isinstance(result, ListOffsetsResultInfo)
        #     assert (result.offset == 1)
```

Reply:
In `test_user_scram_credentials` with KRaft it's giving an error if more than one `UserScramCredentialUpsertion` is given for the same user, even if it's for a different SCRAM mechanism, so you can add an _if_ an do the upsertion in two steps when using kraft

Reply:
> In test_list_offsets we need to disable this check at the moment

This test is passing. Why do we need to disable this test?

>In test_user_scram_credentials with KRaft it's giving an error if more than one UserScramCredentialUpsertion is given for the same user, even if it's for a different SCRAM mechanism, so you can add an if an do the upsertion in two steps when using kraft

Why is it behaving differently in KRaft and Zookeeper? This is a bug in Kafka I think.